src/zip_code_tree.hpp

```diff
@@ -452,6 +452,16 @@ class ZipCodeTree {
      * 
      * New cyclic snarl handling flowchart: 
      * https://docs.google.com/drawings/d/1diKMXCuMteR06fF64BhSttsD5RHh3eTnd8FtyYqs17Y/edit?usp=sharing
+     * 
+     * In order to de-duplicate work, this iterator will not actually look at
+     * _everything_ before it; if it knows that seeds from a given chain would
+     * have already output transitions to its own position. This occurs in
```

Comment:
There's a missing "then" for this "if" I think. I guess it's "then it will skip the chain"?